.gitignore

```diff
@@ -394,4 +394,7 @@ experimental/generator-dotnet-yeoman/node_modules
 .pnp.*
```

Comment:
Do we even need this generator any more? I reckon we can delete it at this point.

Reply:
This would match files that may be generated by yarn - see https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored for details on gitignore.